packages/monitoring-system/data-processor/test/unit/data-processors/cloud-logs-data-processor.spec.ts

```diff
@@ -444,53 +630,54 @@ describe('Cloud Logs Processor', () => {
           firestore: mockFirestore,
           subscription: mockSubscription,
           logger: getMockLogger(mockWriteStream),
+          listenLimit: LISTEN_LIMIT_SECS,
         });
       });
 
       it('logs error for malformed execution start logs', () => {
         return testMalformedMessage(
           loadFixture(['cloud-logs', 'execution-start-missing-id.json']),
-          'Detected malformed execution start logs',
+          'Detected malformed log entry',
```

Comment:
Instead of including the log type in the message, I changed it to be generic and instead I'll include the entire pubsub message in the logs for better debugging